### PR TITLE
AM2R: update FAQ with differences between built in rando

### PR DESCRIPTION
**Pull request by JeffGainsNGames**

### CODE REVIEW

**Comment by JeffGainsNGames** [comment]
Realize I should probably add a WIP to multiworld support until it's implemented.

File: randovania/games/am2r/game_data.py [hunk 1]
```diff
@@ -98,6 +98,14 @@ def _exporter():
             "they look different but behave entirely the same as normal. "
             "In a multiworld game, only your own items can be shiny.",
         ),
+        (
+            "Why play Randovania vs the built in randomizer?",
+            "The Randovania implementation provides additional robustness and features beyond the built in randomizer."
+            "\n\n"
+            "Key features supported by Randovania include full item logic, Door Lock Rando, and Multiworld support. "
```
Comment (Miepee) [inline randovania/games/am2r/game_data.py:105]:
```suggestion
            "Key features supported by Randovania, include but are not limited to, full item logic, toggleable Tricks, Door Lock Rando, and Multiworld support (WIP). "
```

File: randovania/games/am2r/game_data.py [hunk 2]
```diff
@@ -98,6 +98,14 @@ def _exporter():
...
+            "All metroids now drop items including progression and expansions. This feature pairs with a configurable "
```
Comment (Miepee) [inline randovania/games/am2r/game_data.py:106]:
```suggestion
            "All Metroids now also drop items, including progression and expansions. This feature pairs with a configurable "
```

File: randovania/games/am2r/game_data.py [hunk 3]
```diff
@@ -98,6 +98,14 @@ def _exporter():
...
+            "amount of DNA for entrance to the Queen and a hint system (no more mandatory fighting of all metroids). ",
```
Comment (Miepee) [inline randovania/games/am2r/game_data.py:107]:
```suggestion
            "amount of DNA for entrance to the Queen and a hint system (no more mandatory fighting of all Metroids).",
```

**Review by Miepee — APPROVED** [review]

**Pull request merged**